.github/labels.yml

```diff
@@ -63,6 +63,12 @@
 - name: kind/problem
   color: c92712
   description: "Kind: Problem"
+- name: kind/Investigation
```

Comment:
1. How is this different to a spike?
2. Please follow capitalisation for label names: small case, with hyphens for spaces.

Reply:
I can see that a spike and investigation are similar. To my way of thinking a spike is a surgically specific look into an problem or question. eg Look into whether InfluxDB or ElastiSearch works better with Docker Runner.

An investigation would be more general and a bit more time consuming. eg…  Look for the best open-source time series database